.github/CODEOWNERS

```diff
@@ -621,3 +621,4 @@
 /singlestoredb_cloud/*metadata.csv     @tdasarathan @DataDog/ecosystems-review
 /singlestoredb_cloud/manifest.json     @tdasarathan @DataDog/ecosystems-review
 /singlestoredb_cloud/README.md         @tdasarathan @DataDog/ecosystems-review
+/dylibso-webassembly/					         @nilslice @bhelx @DataDog/ecosystems-review
```

Comment:
It looks like the codeowners file was changed back to before. Could you add the codeowners to the files like Singlestore has done:  
![add Dylibso WebAssembly Observe SDK by nilslice · Pull Request #2073 · DataDogintegrations-extras 2023-09-29 at 1 39 06 PM](https://github.com/DataDog/integrations-extras/assets/52013262/a6aafe2d-44e1-435c-96a4-45f449395e21)

Reply:
Ok, done!

Reply:
Looks like it's still failing -- no clue what this should look like as it already appears to be in the same format as other integrations here?